src/renderer/App.vue

```diff
@@ -160,8 +160,8 @@ export default {
     background () {
       return this.$store.getters['session/background'] || `wallpapers/${this.hasSeenIntroduction ? 1 : 2}Default.png`
     },
-    hasAnyProfile () {
-      return !!this.$store.getters['profile/all'].length
+    hasProfile () {
```

Comment:
`is` and `has` should really return a boolean

Reply:
@alexbarnsley do you prefer to use `isProfile()` instead of `hasProfile()` ?

Reply:
I think `hasProfile` is better in this context but it just needs to be a boolean